Addressing "Missing" Issues in Player Guide Full (#127)
DeckofAdventures/TheGame: docs/src/1_Mechanics/01_PlayerGuide_Full.md

1. **Movement** By default, character speed is equal to 6, but may be faster or slower.
Speed is the total number of spaces a character can move in a turn. This total may be 

> split to occur before, after, and/or between actions. A creature can move through a friendly creature's space as part of their movement but can not end their turn on that space. A creature cannot move through a hostile creature's space or between two hostile creatures' spaces if they are adjacent.

Reviewer: Is diagonal adjacent?

Author: Yes, diagonal would be considered adjacent. The below show examples of adjacent in the first two, where diagonal and horizontal/vertical both count as adjacent. 

![example-2-adjacent](https://user-images.githubusercontent.com/54295379/186906341-5fed2c5c-35de-44c1-84c7-aa8fa79ae3d8.png)
![example-1-adjacent](https://user-images.githubusercontent.com/54295379/186906343-00c40d71-4ef1-4425-a8b2-b2e621a170d6.png)

This example shows creatures who are not adjacent.
![example-3-not-adjacent](https://user-images.githubusercontent.com/54295379/186906345-21c20ad1-5833-472b-92ef-21674bf7f9e5.png)

Reviewer: In that case, I think we either need to define adjacent somewhere or just change this to specify 'within one space' - it's clunky, but unambiguous
```suggestion
split to occur before, after, and/or between actions. A creature can move through a friendly creature's space as part of their movement but can not end their turn on that space. A creature cannot move through a hostile creature's space or between two hostile creatures' spaces if they are within one space of one another.
```

Reviewer 2: I'm still a bit confused after seeing/reading this. Where can my character not walk through in the examples above. I know I can't walk through an enemy's space. This is adding that I can't walk past 2 enemies if they're next to each other? @CBroz1

Reviewer: Exactly, you'd have to go around 2 enemies next to each other

Author: Agreed with @CBroz1 on adjusting language away from Adjacent - better to be too specific than not enough. Since we don't allow for reaction-based attacks of opportunity I think allowing enemies (and allies) to create a 'line' that can't be passed allows for some form of similar battlefield control.

In the first two examples, if there were walls on either side of the set of enemies you would need to defeat one of them in order to move past rather than being able to move 'through' the space that they control by being within 1 space of each other. 

This also allows in a narrow 2-space wide hallway a Defender and Martial to stand side by side and prevent the goblins from rushing the Support who stands 3 spaces farther back down the hallway.

        - *Hide*: Stealth vs. target(s) Detection; impose Lower Hand on attacks against
         you by target(s) who failed

#### Ending Combat

> When the last enemy combatant has been Knocked Out, retreats, or surrenders, Combat is over. This ends the turn order, and characters can act independently of turn order. Characters do not discard their Target Cards after combat ends. This TC represents the character until the next combat is initiated, at which point a new TC is drawn. 

Reviewer: ```suggestion
When the last enemy combatant has been Knocked Out, retreats, or surrenders, Combat is over. This ends the turn order, and characters can act independently of turn order. Characters do not discard their Target Cards after combat ends. This TC represents the character until the start of the next combat. 
```

3. **Knocked Down** A Dealer must expend a Minor Action to get up. Their movement speed
is halved, rounding up. All Strength and Agility Checks are made with Lower Hand. When
Knocked Down, attackers within 1 space draw with Upper Hand and attackers further than

> 1 space away draw with Lower Hand. A creature can willingly go prone and be considered Knocked Down as part of their movement.

Reviewer: ```suggestion
1 space away draw with Lower Hand. A creature can spend 1 speed to willingly go prone as part of their movement, gaining the Knocked Down status.
```

|   5   |      6      |   3   |
|   8   |      8      |   4   |

### Interacting with the Environment

#### Light and Vision

Certain types of Checks require light and vision to see the environment or a target
effectively. The amount of light available impacts Checks for most creatures. There are some Powers that affect a creature's abilities depending on the amount of light available. 

**Partial Darkness:** When making a Detection or Investigation Check in partial

>   darkness, draw with the Lower Hand.
> 
> *Examples of Partial Darkness:*
> - A dark alley in a city with some light spilling over from a nearby street. 
> - An underground room with some candles burning lightly. 

Reviewer: ```suggestion
  darkness, draw with the Lower Hand. For example...
    - A dark alley in a city with some light spilling over from a nearby street. 
    - An underground room with some candles burning lightly. 
```

> **Full Darkness:** When making a Check in full darkness, draw with the Lower Hand.
> 
> *Examples of Full Darkness:* 
> - A moonless night in the countryside without any light source. 
> - Descending into a cave without a light source.

Reviewer: Does this include knowledge checks? 
> Murder mystery. The lights go out. I believe the killer is in the room. I think I smell their perfume. Is that lavender? Draw knowledge check

What about Vitality checks? Do I bleed worse in the dark?

Maybe we change to same as above but lower hand 3
```suggestion
**Full Darkness:** When making any Check in full darkness, draw with the Lower Hand. For example ...
    - A moonless night in the countryside without any light source. 
    - Descending into a cave without a light source.
```

Author: I adjusted to be Lower Hand (3) on Detection or Investigation. With that adjustment Darkness does not affect Attacks, meaning in a pitch black room you can still aim perfectly at a nearby target (or one up to X feet away based on ranged attacks.) 

Might be worth us revisiting in playtesting if we encounter Darkness often and want to adjust the intensity of its effects, but resolving for now.

Reviewer: No objection, but I'm thinking maybe attacks should also lower hand (2 or 3) in full darkness.

#### Cover 

When a creature moves to a position that makes it more difficult to hit them, they will
gain the effects of cover.

**Partially Covered**: When a creature is still visible behind something but it is still
  reasonable to hit them. While partially covered the creature gains +1 AR. 

*Examples of Partially Covered*:
- Standing directly between another creature and an attacker.
- Standing behind a crate or barrel that is half the creature's height. 

**Fully Covered**: when a creature is nearly impossible to see behind something and it
  is unlikely to land a hit on them. While fully covered, the creature gains +2 AR.

*Examples of Fully Covered*: 
- Crouching below a crate that is half the creature's height.
- Standing behind a pillar or wall that is taller than the creature. 

A creature can go from Fully to Partially Covered or from Fully Covered to Not Covered
by using their Speed or making a Check. Some Powers may grant these effects as well. A
creature may be covered from one direction but not from another direction. 

#### Falling

Falling from a great height is dangerous. At the end of a fall, a creature takes 1

> damage and loses 1 speed for every 10 feet it fell and the creature is Knocked Down.
> This speed is recovered after a Rest or if the creature is healed for at least 1 HP.

Reviewer: ```suggestion
damage and applies -1 speed for every 10 feet it fell and the creature is Knocked Down.
This speed is recovered after a Rest or if the creature is healed to their maximum HP.
```

#### Challenging Terrain

Moving through the environment can sometimes be challenging. When navigating

> through *Challenging Terrain*, a creature moves at half speed, meaning moving through
> one space requires 2 speed. While standing in Challenging Terrain, all Agility Checks
> are made with the Lower Hand. 

Reviewer: ```suggestion
through *Challenging Terrain*, moving through
one space requires 2 speed. While standing in Challenging Terrain, all Agility Checks
are made with the Lower Hand. 
```

gain the effects of cover.

**Partially Covered**: When a creature is still visible behind something but it is still
  reasonable to hit them. While partially covered the creature gains +1 AR. 

*Examples of Partially Covered*:
- Standing directly between another creature and an attacker.
- Standing behind a crate or barrel that is half the creature's height. 

**Fully Covered**: when a creature is nearly impossible to see behind something and it
  is unlikely to land a hit on them. While fully covered, the creature gains +2 AR.

*Examples of Fully Covered*: 
- Crouching below a crate that is half the creature's height.
- Standing behind a pillar or wall that is taller than the creature. 

A creature can go from Fully to Partially Covered or from Fully Covered to Not Covered
by using their Speed or making a Check. Some Powers may grant these effects as well. A
creature may be covered from one direction but not from another direction. 

#### Falling

Falling from a great height is dangerous. At the end of a fall, a creature takes 1
damage and applies -1 speed for every 10 feet it fell and the creature is Knocked Down.

> This speed is recovered after a Rest or if the creature is healed to their maximum HP.

Reviewer: comment to generate link
